README.md

```diff
@@ -104,6 +104,7 @@ __Install SuperBench__
 
     # install superbench
     python3 -m pip install .
+    ansible-galaxy collection install ansible.utils community.crypto
```

Comment:
If the command is necessary for SB, I think we should package the command, just as you said python setup.py xxxx

Reply:
use `make postinstall` instead